### ISSUE TITLE
Imbalanced learning simple solution 

### ISSUE THREAD

**Issue by MaxHalford** [creme-ml/creme#2]
It would be nice to see if we could implement something simple to handle imbalanced learning. This could be part of a new sub-module called `imblearn` in reference to the [`imbalanced-learn
`](https://github.com/scikit-learn-contrib/imbalanced-learn) library.

**Comment by AdilZouitine** [comment]
I'm on it and I'm proposing two things:
- Weighted cross-entropy
- [The focal loss](https://arxiv.org/pdf/1708.02002.pdf)

**Comment by MaxHalford** [comment]
You can add a weight parameter to `LogLoss` too, no?

**Comment by AdilZouitine** [comment]
Yes :)

**Comment by MaxHalford** [comment]
We'll need to find a dataset and do a comparison with and without the new losses.

**Comment by AdilZouitine** [comment]
- [Credit Card Fraud Detection](https://www.kaggle.com/mlg-ulb/creditcardfraud/home) the dataset is high imbalanced, with only 0.17% of transactions being classified as fraudulent

It seems like a good data set.

**Comment by MaxHalford** [comment]
Yep it's perfect! Let's add it to the `datasets` module.

**Comment by MaxHalford** [comment]
@AdilZouitine I added weight parameters to `losses.Log` and `losses.CrossEntropy`. Once we add `losses.Focal` we can close this issue :)
